src/MockRealitio.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.8.0;
+
+interface IRealitio {
+    function getContentHash(bytes32 questionId) external view returns (bytes32);
+    function getOpeningTS(bytes32 questionId) external view returns (uint32);
+    function resultFor(bytes32 questionId) external view returns (bytes32);
+}
+
+contract MockRealitio is IRealitio {
+    mapping(address => bool) public owners;
+    mapping(bytes32 => bytes32) private contentHashes;
+    mapping(bytes32 => uint32) private openingTimestamps;
+    mapping(bytes32 => bytes32) private results;
+
+    constructor() {
+        owners[msg.sender] = true;
+    }
+
+    modifier onlyOwners() {
+        require(owners[msg.sender], "Only owner can call this function");
+        _;
+    }
+
+    function addOwner(address newOwner) external onlyOwners {
+        owners[newOwner] = true;
+    }
+
+    function removeOwner(address ownerToRemove) external onlyOwners {
+        require(msg.sender != ownerToRemove, "Owner cannot remove themselves");
+        owners[ownerToRemove] = false;
+    }
+
+    function getContentHash(bytes32 questionId) external view returns (bytes32) {
+        return contentHashes[questionId];
```

Comment:
These getters aren't failing when called with `questionId`s that don't exist and will rather return 0-values. Is this what is expected by the interface?

If not, adding this caveat as a comment would be sufficient, as this is a mock implementation.

Reply:
Yeah, this is expected; see https://github.com/RealityETH/reality-eth-monorepo/blob/f786d5f32cbe7e8f0c5af7f019eb4b3accc7fa5e/packages/contracts/development/contracts/RealityETH-4.0.sol#L667 .